src/services/measure/measure.js

```diff
@@ -0,0 +1,278 @@
+goog.provide('ngeo.interaction');
+goog.provide('ngeo.interaction.Measure');
+
+goog.require('goog.dom');
+goog.require('goog.dom.classlist');
+goog.require('goog.events');
+goog.require('ol.DrawEvent');
+goog.require('ol.Feature');
+goog.require('ol.FeatureOverlay');
+goog.require('ol.MapBrowserEvent');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.interaction.Pointer');
+goog.require('ol.style.Style');
+
+
+
+/**
+ * @classdesc
+ * Interaction that allows measuring (length, area, ...).
+ *
+ * @constructor
+ * @extends {ol.interaction.Pointer}
+ * @param {{startMsg: (string|undefined),
+ *          style:(ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)}
+ *         =} opt_options Options
+ */
+ngeo.interaction.Measure = function(opt_options) {
+
+  var options = goog.isDef(opt_options) ? opt_options : {};
+
+  /**
+   * The help tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.helpTooltipElement_ = null;
+
+
+  /**
+   * Overlay to show the help messages.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.helpTooltip_ = null;
+
+
+  /**
+   * The measure tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.measureTooltipElement_ = null;
+
+
+  /**
+   * Overlay to show the measurement.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.measureTooltip_ = null;
+
+
+  /**
+   * The sketch feature.
+   * @type {ol.Feature}
+   */
+  this.sketchFeature = null;
+
+  /**
+   * The draw interaction to be used.
+   * @type {ol.interaction.Draw|ngeo.interaction.DrawAzimut}
+   */
+  this.drawInteraction = null;
+
+  /**
+   * The message to show when user is about to start drawing.
+   * @type {string}
+   */
+  this.startMsg = goog.isDef(options.startMsg) ? options.startMsg :
+      'Click to start drawing';
```

Comment:
Do you already know how we'll handle internationalization?

If the user messages are of type `(string|Element)` we may be able to pass `$compile`'d messages to the interaction.

Something for a separate PR maybe, but worth considering I think.